.github/workflows/checklist.yml

```diff
@@ -0,0 +1,99 @@
+name: Checklist
+
+# Produce a list of things that need to be changed
+# for the submission to align with CONTRIBUTING.md
+
+on:
+  pull_request:
+    types: [ opened, reopened, edited, synchronize ]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  checklist:
+    name: commit
+    runs-on: ubuntu-latest
```

Comment:
`# XXX would prefer to run this on FreeBSD-latest` :)

Reply:
Yes, hopefully one day